elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -133,6 +133,24 @@ class PfeContentSet extends PFElement {
     super.connectedCallback();
 
     if (this.hasLightDOM()) {
+      this.setAttribute("hidden");
+
+      // If the tab does not exist in the light DOM, add it
+      if (!this.tabs) {
+        let newEl = document.createElement("pfe-tabs");
+        newEl.setAttribute("hidden", "");
```

Comment:
So the theory is that we set everything to hidden and then let the `_toggleVisible` function add or remove those attributes. Do I have that correct? :)

Reply:
Toggle visible is only impacting the visibility of the tab and accordion. The hidden on content set is to prevent the flash before upgrade of the light DOM data.